Server/controllers/Bill/getUserBillType.js

```diff
@@ -0,0 +1,23 @@
+const {
+  successMessage,
+  internalErrorMessage,
+} = require('../../helpers/responseMessage');
+
+const { getUserBillType } = require('../../database/query/Bills');
+
+module.exports = (req, res) => {
+  //const  {id}  = req.user => it comes from miderware after user login
+
```

Comment:
Why you have it in the URL then:` /api/user/:userId/home`

Reply:
it change to `/api/home`